Misc/NEWS.d/next/Build/2022-06-20-15-15-11.gh-issue-90656.kFBbKe.rst

```diff
@@ -0,0 +1 @@
+Add new triplets for loongarch64
```

Comment:
This, and the PR title, could probably be "Add platform triplets for LoongArch64" -- shorter and properly capitalized. (Things just added are implicitly "new".)